package.json

```diff
@@ -34,12 +34,13 @@
   "dependencies": {
     "@stablelib/random": "^1.0.1",
     "cids": "^1.1.6",
```

Comment:
@stephhuynh18 `cids` package is still listed in the dependencies list. Mind removing it, please? If I understand right, it should not longer be required.

Reply:
👍